flask/sessions.py

```diff
@@ -348,6 +349,8 @@ class SecureCookieSessionInterface(SessionInterface):
     """The default session interface that stores sessions in signed cookies
     through the :mod:`itsdangerous` module.
     """
+    #: the regular expression used to split the value of the 'Vary' header
+    comma_delim_header_re = re.compile(r'\s*,\s*')
```

Comment:
Doesn't belong here IMO, it's not meant to be overridable. I think a somewhat common place to store regexes like this is an underscored function arg since they are evaluated only once and do not involve a lookup on the module scope, i.e.

    def _patch_vary_cookie_header(self, response, _comma_re=re.compile(r'\s*,\s*')):